app/Models/Attendance.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Attendance extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'totalhours',
+    ];
+
+    public function users()
+    {
```

Comment:
This relationship isnt quite right. One attendance entry belongs to a single user
`public function user()
{

return $this->belongsTo(User::class)

}`
You also need to add the reverse relationship of this on the users model. A user can have many attendance entries

`public function attendance()
{

return $this->hasMany(Attendance::class)

}`